common/app/common/TrailsToShowcase.scala

```diff
@@ -330,7 +318,16 @@ object TrailsToShowcase {
   private def webUrl(content: PressedContent): Option[String] =
     content.properties.maybeContent.map(_.metadata.webUrl)
 
-  private def titleFrom(content: PressedContent): String = content.header.headline
+  def titleOfLengthFrom(length: Int, content: PressedContent): Either[Seq[String], String] = {
+    val trailTitle = TrailsToRss.stripInvalidXMLCharacters(content.header.headline)
+    Some(trailTitle)
```

Comment:
Might be nicer as:
```
Right(trailTitle)
  .filterOrElse(_.nonEmpty, Seq(s"The headline '$trailTitle' is empty")
  .filterOrElse(_.length <= length, Seq(...))
```

Reply:
.filter on Either. Interesting going to try this on an upstream PR.